docs/authentication/social-connections/custom-provider.mdx

```diff
@@ -0,0 +1,78 @@
+---
+description: Configure a custom OpenID Connect (OIDC) compatible authentication provider for your Clerk application.
+---
+
+# OAuth with a custom auth provider
+
+<Callout type="warning">
+  Custom auth providers are in beta. Please [contact support](/contact/support) to get your application whitelisted for
+  this feature.
+</Callout>
+
+Clerk allows you to configure custom OpenID Connect (OIDC) compatible authentication providers for your application. This guide walks you through the steps to set up a custom OAuth provider.
+
+## Prerequisites
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "A Clerk application is required.",
+      link: "/docs/quickstarts/setup-clerk",
+      icon: "clerk",
+    },
+    {
+      title: "An OIDC identity provider is required.",
+      link: "https://openid.net/specs/openid-connect-core-1_0.html",
+      icon: "user-circle",
+    }
+  ]}
+>
+  - Configure custom Auth provider for your Clerk application.
+</TutorialHero>
+
+## Configuration
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections)
+1. In the navigation sidebar, select **User & Authentication > Social Connections**.
+1. Under **Custom Auth Providers**, select **Add provider**.
+1. Fill in the following fields:
+
+- **Name**: The provider name (visible to users)
+- **Key**: A unique identifier for the provider (cannot be changed after creation)
+- **Discovery Endpoint**: The OIDC discovery endpoint URL of your provider
+- Alternatively, select **Manual configuration** to set up provider endpoints manually
```

Comment:
Indent this bullet point!
so that the bullet point is shown to be related to "Discovery Endpoint"

```suggestion
  - Alternatively, select **Manual configuration** to set up provider endpoints manually
```

Reply:
That makes sense. However, it doesn't seem to be supported by the CSS or something, as the preview doesn't have this indented. But let's keep it as is 👍🏼

Reply:
Ah, actually it was like this, and that's why I removed the indentation.

<img width="737" alt="image" src="https://github.com/user-attachments/assets/dc078034-a419-445d-b8d7-3e999111a897">

@bradlc is this a known issue maybe?